extensions/devservices/oracle/src/main/java/io/quarkus/devservices/oracle/deployment/OracleDatasourceServiceConfigurator.java

```diff
@@ -36,7 +36,7 @@ public RunningDevServicesDatasource composeRunningService(ContainerAddress conta
     }
 
     public String getJdbcUrl(ContainerAddress containerAddress, String databaseName) {
-        return "jdbc:%s://@%s:%d/%s%s".formatted(
+        return "jdbc:%s:@//%s:%d/%s%s".formatted(
```

Comment:
Are we sure about this? Because it's not in line with what we had before and it was working fine given nobody complained.

What we had before is: https://github.com/quarkusio/quarkus/commit/45d4db21c224cc9c9a0811b130e1de94fd51cb00 .

We really need to be extra sure.

Reply:
I've checked here : https://docs.oracle.com/en/database/oracle/oracle-database/21/jajdb/oracle/jdbc/OracleDriver.html and on the intellij connection console. But I just saw that OracleContainer in test containers return URL without the `//`, so I don't think it makes a difference.

Reply:
I agree that it looks weird. I removed the `//` for now, let's keep with that